cmd/google.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	googleCmd = &cobra.Command{
+		Use:      "google",
+		Short:    "Terracognita reads from Google and generates hcl resources and/or terraform state",
+		Long:     "Terracognita reads from Google and generates hcl resources and/or terraform state",
```

Comment:
reads from GCP, I'm aware Google is the name of the provider, but the service we read from is GCP

Reply:
Yep in this case it's better